packages/plugin-e2e/src/api.ts

```diff
@@ -15,7 +15,19 @@ import { grafanaE2ESelectorEngine } from './selectorEngine';
 import { ExplorePage } from './models/ExplorePage';
 
 export type PluginOptions = {
-  selectorRegistration: void;
```

Comment:
is removing this from the type intentional?

Reply:
Good catch - should have added a GH comment about this. Yes, it was intentionally removed. It's not an actual fixture option, so it doesn't belong there.